packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -1355,15 +1355,15 @@ model = SwarmFormerModel.from_pretrained("${model.id}")
 
 const mlx_unknown = (model: ModelData): string[] => [
 	`# Download the model from the Hub
-pip install huggingface_hub hf_transfer
+pip install huggingface_hub[hf_xet]
 
-export HF_HUB_ENABLE_HF_TRANSFER=1
+export HF_XET_HIGH_PERFORMANCE=1
```

Comment:
I'd suggest we remove this to make everything simpler. In my experience it works fast enough without the flag.